feature-libs/asm/schematics/add-asm/schema.json

```diff
@@ -15,6 +15,11 @@
       "type": "boolean",
       "description": "Lazy load the asm features.",
       "default": true
+    },
+    "configuration": {
```

Comment:
Feels like an overkill if we only want to show warning (and still install b2b config if it is not present). Maybe instead we will just in feature prompt for b2b features after each feature put in parethesis info: `(b2b feature)`?

Reply:
It's needed for propagating the configuration choice from the projects/schematics prompt.
For ASM it might not be needed, but for organization it's needed for sure.